pkg/chains/signing/wrap.go

```diff
@@ -117,7 +117,7 @@ func (w *sslSigner) SignMessage(payload io.Reader, opts ...signature.SignOption)
 	if err != nil {
 		return nil, err
 	}
-	env, err := w.wrapper.SignPayload(in_toto.PayloadType, m)
+	env, err := w.wrapper.SignPayload(context.TODO(), in_toto.PayloadType, m)
```

Comment:
Can we make `SignMessage` take a context? It looks like the only place that calls this method is from `pkg/chains/signing.go:Sign` which does have a context.

(Ok to do as a follow up)